app/views/scoreboards/_table.html.erb

```diff
@@ -18,7 +18,12 @@
           </td>
           <td>
             <h2 class="nowrap">
-              <%= link_to school.sum_points, school_timeline_path(school, academic_year: @academic_year), class: 'badge badge-success' %>
+              <%= link_to school.sum_points,
+                          school_timeline_path(
+                            school, (
+                              { academic_year: @academic_year } unless @scoreboard.national?)
```

Comment:
wonder if end bracket should be on next line? or everything on the same line?

Reply:
I will have another go but this is what I ended up with after a small fight with erblint :-)